telosnetwork/open-block-explorer
# replacing pug for html

# Fixes #585

## Description
This PR replaces all pug code for the equivalent in HTML 

## Test scenarios
- Open the [current testing site](https://explorer-test.telos.net/)
- Open [this PR testing link](https://deploy-preview-627--obe-staging.netlify.app/) another browser tab
- Browse whenever you want but do it in both at the same time
- You should not notice any difference
